kotlin-native/Interop/Runtime/src/native/kotlin/kotlinx/cinterop/Vector128.kt

```diff
@@ -57,10 +57,25 @@ public final class Vector128 private constructor() {
 @GCUnsafeCall("Kotlin_Interop_Vector4f_of")
 public external fun vectorOf(f0: Float, f1: Float, f2: Float, f3: Float): Vector128
 
+@SinceKotlin("1.9")
```

Comment:
The argument doesn't seem right -- this declaration won't appear in Kotlin 1.9. See more details here: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-since-kotlin/